tutorials/multimodal/NeVA Tutorial.ipynb

```diff
@@ -2,8 +2,13 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "id": "a2225742c5996304",
-   "metadata": {},
+   "id": "672caa4e",
+   "metadata": {
+    "collapsed": false,
+    "jupyter": {
+     "outputs_hidden": false
+    }
+   },
    "source": [
     "# NeVA Training / Inference Tutorial\n",
```

Comment:
This line doesn't show in the code diff, but I think you can put this script:
`docker run --gpus all -it --rm -v <your_nemo_dir>:/opt/NeMo --shm-size=8g \ ...`
into a code snippet, instead of just a text box.